_posts/en/newsletters/2020-09-02-newsletter.md

```diff
@@ -0,0 +1,252 @@
+  transaction; the adaptor can only be transformed into Bob's signature
+  if that signature will reveal one of Alice's private keys to Bob.  The
+  other private key Alice created is the revocation private key, which
+  she'll give to Bob if she wants to revoke this channel state in favor
+  of a later state.  Similarly, Bob generates his own keys and receives
+  a signature adaptor from Alice.
+
+  In this way, even though the commitment transaction is the same for
+  both parties, there will be a difference in the signatures (called
+  *witnesses*) each of them uses to unilaterally close the channel.
+  The difference in the signatures will give the party who didn't close
+  the channel a chance to create and broadcast a penalty transaction if
+  the channel was closed in an old state.  Otherwise, if the channel was
+  closed in its latest state, both parties get their money, just like in
+  the current LN.
+
+  The proposal also suggests using basically the same scheme to
+  transition from Hash Time Lock Contracts (HTLCs) to Point Time Lock
+  Contracts ([PTLCs][news92 ptlcs]), which is already a goal several LN contributors
+  have voiced support for in the past.
+
+  Advantages of this approach are that it might require less private
+  data storage (though this is [debated][zmn reply]), it makes unilateral closes
+  a bit smaller onchain, and it describes a path to using PTLCs that
+  will provide LN users with additional privacy against routing
+  node surveilance as well as bring the protocol a little closer to what it may
+  look like when it can take advantage of the benefits of [schnorr
+  signatures][topic schnorr signatures].
+
+- **Default signet discussion:** Michael Folkson [posted][default signet post] to the
+  Bitcoin-Dev mailing list a question about whether there should be a
+  default [signet][topic signet] similar to Bitcoin's existing default testnet.
+  Signets are test networks where valid blocks must be
+  signed by one or more trusted keys; this allows them to be more
+  predictable for testing than fully decentralized test networks that
+  are often subject to vandalism.  [BIP325][] describes the signet
+  protocol, which anyone can use to create their own signet, but there
+  is a [desire][bitcoin core default signet] to include a default signet into
+  Bitcoin Core (and possibly other software) that
+  users will be able to enable with a single configuration option.
+  Folkson's first question is whether anyone has any objection to this.
+
+    If there is to be a default signet, Folkson posted several
+    additional questions about how it should be administered.  For
+    example, who should control its signing keys?  How many keys should
+    there be?  Under what circumstances should the default signet be
+    reset?  When should proposed consensus changes to mainnet Bitcoin be
+    implemented by the trusted signet miners/signers?
```

Comment:
is `miners` correct / necessary here?

Reply:
I think "signers" by itself might be ambiguous, as the way we usually use that term only applies to the people who sign transactions.

Reply:
"by the trusted signet block signers and miners"

Miners aren't trusted, only the block signers

Reply:
If the miners don't upgrade when the signet block signers do they risk being forked off the Signet chain. So for the sake of the network it isn't necessary that miners upgrade but they will need to follow the block signers in upgrading to continue mining blocks on the Signet chain.

Reply:
in line 80-81 is explained that there are block signers in signet, no miners.
I think the user will have this still in mind while reading the end of this paragraph.

fwiw, I got confused with the mentioning of miners...

Reply:
ACK to `by the trusted signet block signers.`

Reply:
If the miners are following the block signers then yeah perhaps get rid of "miners" from this sentence.

"by the trusted signet block signers"